integration_tests/test_client_integrated.py

```diff
@@ -361,6 +361,29 @@ def test_processpools_parallelfuncs(
         mpPool=poolclass)
 
 
+@pytest.fixture(scope='module')
+def example_renderparameters(render, teststack):
+    zvalues = render.run(renderapi.stack.get_z_values_for_stack, teststack)
+    z = zvalues[0]
+    bounds = render.run(renderapi.stack.get_bounds_from_z, teststack, z)
+    width = (bounds['maxX'] - bounds['minX']) / 2
+    height = (bounds['maxY'] - bounds['minY']) / 2
+    x = bounds['minX'] + width / 4
+    y = bounds['minY'] + width / 4
+    renderparams = renderapi.image.get_bb_renderparams(
+        teststack, z, x, y, width, height,
+        scale=0.25, render=render)
+    yield renderparams
+
+
+@pytest.fixture(scope='module')
+def renderedimg_renderparams(render, example_renderparameters):
+    # TODO is there a "render from renderparams" api?
+    arr = renderapi.image.get_renderparameters_image(
```

Comment:
I think the api on render side is just called "render" in the sense that rendering from render_parameters was sort of the core of things.  I would argue we might want to call this renderapi.image.render(renderparameters)

Reply:
Are you refering to the "TODO" note here?  I added that api call but forgot to remove the comment.